### PR TITLE
Wizard recipe: GLU-v9.0.1

### PR DESCRIPTION
**Pull request by Keno**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: GLU
* Version: v9.0.1

@staticfloat please review and merge.

### CODE REVIEW

File: G/GLU/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+cd libglu
+./autogen.sh --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j ${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux i686 {libc=glibc},
```
Comment (Keno) [inline G/GLU/build_tarballs.jl:24]: I'm thinking the platform refactor broke some printing here.

Comment (giordano) [reply G/GLU/build_tarballs.jl:24]: I have the same gripe 🙂

Comment (giordano) [reply G/GLU/build_tarballs.jl:24]: And I didn't think this badly breaks the wizard, that's bad

File: G/GLU/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    Linux x86_64 {libc=glibc},
+    Linux aarch64 {libc=glibc},
+    Linux armv7l {call_abi=eabihf, libc=glibc},
+    Linux powerpc64le {libc=glibc},
+    Linux i686 {libc=musl},
+    Linux x86_64 {libc=musl},
+    Linux aarch64 {libc=musl},
+    Linux armv7l {call_abi=eabihf, libc=musl},
+    FreeBSD x86_64
+]
```
Comment (giordano) [inline G/GLU/build_tarballs.jl:34]:
```suggestion
platforms = filter(p -> Sys.islinux(p) || Sys.isfreebsd(p), supported_platforms())
```

File: G/GLU/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    GitSource("https://salsa.debian.org/xorg-team/lib/libglu.git", "d77f0cae59ce18bc7bba7b1f0c0b605224c23783")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd libglu
+./autogen.sh --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j ${nproc}
+make install
+"""
```
Comment (giordano) [inline G/GLU/build_tarballs.jl:19]:
```suggestion
install_license debian/copyright
"""
```

**Pull request merged**